inst/examples/iconList.R

```diff
@@ -0,0 +1,66 @@
+library(leaflet)
+
+# adapted from http://leafletjs.com/examples/custom-icons.html
+
+iconData = data.frame(
+  lat = c(rnorm(10, 0), rnorm(10, 1), rnorm(10, 2)),
+  lng = c(rnorm(10, 0), rnorm(10, 3), rnorm(10, 6)),
+  group = rep(sort(c('green', 'red', 'orange')), each = 10),
+  stringsAsFactors = FALSE
+)
+
+leaflet() %>% addMarkers(
+  data = iconData,
+  icon = ~ iconList(
+    iconUrl = sprintf('http://leafletjs.com/docs/images/leaf-%s.png', group),
+    shadowUrl = 'http://leafletjs.com/docs/images/leaf-shadow.png',
+    iconSize = list(c(38, 95)), shadowSize = list(c(50, 64)),
+    iconAnchor = list(c(22, 94)), shadowAnchor = list(c(4, 62)),
+    popupAnchor = list(c(-3, -76))
```

Comment:
Let's try to stay away from lists of x-y or lat-lng vectors. Either `iconAnchorX/iconAnchorY` or add a helper function that takes x and y or lat and lng.